rudder-web/src/main/webapp/javascript/rudder/homePage.js

```diff
@@ -37,10 +37,17 @@ function homePage (
   , globalGauge
   , nodeCompliance
   , nodeComplianceColors
+  , pendingNodes
 ) {
   $("#globalCompliance").append(buildComplianceBar(globalCompliance));
   createTooltip();
 
+
+  var stats = "Compliance based on "+ pendingNodes.active + " Nodes."
+  if (pendingNodes.pending !== null) {
+    stats += "There is also " + pendingNodes.pending.nodes + " pending Nodes (" + pendingNodes.pending.percent + "% of total Nodes)"
```

Comment:
not pending nodes :
There is also XX nodes for which we are still wainting for data (YY%)